podpac/core/node.py

```diff
@@ -424,8 +424,16 @@ def _base_definition(self):
 
             if (
                 isinstance(value, Node)
-                or (isinstance(value, (list, tuple, np.ndarray)) and all(isinstance(elem, Node) for elem in value))
-                or (isinstance(value, dict) and all(isinstance(elem, Node) for elem in value.values()))
+                or (
+                    isinstance(value, (list, tuple, np.ndarray))
+                    and (len(value) > 0)
```

Comment:
Added this part of the if statement. Doesn't to any damage and helps with the node_probe implementation.

Reply:
If this is not present, empty lists and dictionaries end up in the "inputs" field.